### PR TITLE
DO NOT MERGE -- Jenkins test

### PR DESCRIPTION
**Pull request by mweinelt**
Don't mind me, I'm just a test!

### CODE REVIEW

File: contrib/ci/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+    stages {
+        stage('lint') {
+            steps {
+                sh 'luacheck --std=min {packages,scripts,target}'
+            }
+        }
+        stage('docs') {
+            steps {
+                sh' make -C docs html'
```
Comment (2tata) [inline contrib/ci/Jenkinsfile]: It will be good if the docs can be deployed as well.

Comment (mweinelt) [reply contrib/ci/Jenkinsfile]: That certainly would be awesome, but I'd say lets get this going first before we learn to fly :)

Comment (mweinelt) [reply contrib/ci/Jenkinsfile]: I remember ffrgb/meshviewer was using https://www.netlify.com/ for test deployments.

File: contrib/ci/jenkins-community-slave/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
+FROM gluon
```
Comment (mweinelt) [inline contrib/ci/jenkins-community-slave/Dockerfile]: Fetch from https://hub.docker.com/r/gluonmesh/build

**Comment by mweinelt** [comment]
```
*** site.conf error: expected mesh.vxlan to be a boolean, but it is unset
```

**Pull request closed without merging**